### PR TITLE
social-logos: Replace svgstore-cli dep with svgstore directly

### PR DESCRIPTION
**Pull request by anomiex**
## Proposed changes:

The `svgstore-cli` package is a fairly simple wrapper around the `svgstore` library. We can drop a dep on a deprecated version of `glob` by using the library directly.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

None

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Build artifact should be unchanged.

### CODE REVIEW

File: projects/js-packages/social-logos/tools/svg-to-sprite.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const svgText = sprites
+	.toString()
+	// Make SVG more conformant.
+	.replace( '<svg><defs/>', '<svg xmlns="http://www.w3.org/2000/svg">' );
+
+fs.writeFileSync( destSpriteFilename, svgText, 'utf8' );
+
+// Copy example files.
+fs.cpSync( `${ srcSpriteDir }/.`, `${ destSpriteDir }/.`, { recursive: true } );
```
Comment (tbradsha) [inline projects/js-packages/social-logos/tools/svg-to-sprite.js]: Now there's only one file in here (`example.html`), so perhaps we should just explicitly move that. No need for recursive either.

```suggestion
fs.cpSync( `${ srcSpriteDir }/example.html`, `${ destSpriteDir }/example.html` );
```

File: projects/js-packages/social-logos/tools/svg-to-sprite.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+const svgDir = 'build/svg-clean';
+const srcSpriteDir = 'src/svg-sprite';
+const srcCssDir = 'src/css';
```
Comment (tbradsha) [inline projects/js-packages/social-logos/tools/svg-to-sprite.js]: These are now only used once, so we could probably change them to something like the following:

```
const srcSpriteExampleHTML = 'src/svg-sprite/example.html';
const srcExampleCSS = 'src/css/example.css';
```

File: projects/js-packages/social-logos/tools/svg-to-sprite.js (hunk 1 above)
Comment (anomiex) [reply projects/js-packages/social-logos/tools/svg-to-sprite.js]: Meh. I was matching the sense of what the old code did (without having to manually do the glob and copy file by file).

**Review by tbradsha — COMMENTED** [review]
This is great. I was planning to do the same with `svg-optimize` as well. A few inline comments to consider.

File: projects/js-packages/social-logos/tools/svg-to-sprite.js (hunk 2 above)
Comment (anomiex) [reply projects/js-packages/social-logos/tools/svg-to-sprite.js]: If you want to, feel free to amend the PR. At the moment I'm feeling lazy and not seeing either way as making enough of a difference to be worth the effort when I can't just click "commit suggestion". 😀

File: projects/js-packages/social-logos/tools/svg-to-sprite.js (hunk 1 above)
Comment (tbradsha) [reply projects/js-packages/social-logos/tools/svg-to-sprite.js]: Yeah, the old code was an artifact of an older structure and I should've tightened it up at that point.

File: projects/js-packages/social-logos/tools/svg-to-sprite.js (hunk 2 above)
Comment (tbradsha) [reply projects/js-packages/social-logos/tools/svg-to-sprite.js]: 67214e295f

**Review by tbradsha — APPROVED** [review]

**Pull request merged**